scripts/install.bat

```diff
@@ -0,0 +1,5 @@
+
+
+pipenv install
+pipenv run python -m spacy download en_core_web_sm
+pipenv run python -m spacy download en_core_web_md
```

Comment:
Any status update on the windows script?

Reply:
We are currently working on a version that executes the correct commands with the correct versions, yet we are having some issues where there are some dependencies that error out before installing.